src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -192,6 +226,90 @@ public function get_item_schema() {
 			'title'      => 'theme',
 			'type'       => 'object',
 			'properties' => array(
+				'author'         => array(
+					'description' => __( 'The theme author.' ),
+					'type'        => 'string',
+					'readonly'    => true,
+					'properties'  => array(
+						'raw'      => array(
+							'description' => __( 'The theme author\'s name, as found in the theme header.' ),
+							'type'        => 'string',
+							'readonly'    => true,
+						),
+						'rendered' => array(
+							'description' => __( 'HTML for the theme author, transformed for display.' ),
+							'type'        => 'string',
+							'readonly'    => true,
+						),
+					),
+				),
+				'author_uri'     => array(
+					'description' => __( 'The website of the theme author.' ),
+					'format'      => 'uri',
+					'type'        => 'string',
+					'readonly'    => true,
+					'properties'  => array(
+						'raw'      => array(
+							'description' => __( 'The website of the theme author, as found in the theme header.' ),
+							'type'        => 'string',
+							'readonly'    => true,
+						),
+						'rendered' => array(
+							'description' => __( 'The website of the theme author, transformed for display.' ),
+							'type'        => 'string',
```

Comment:
uri should be here

Reply:
6c0bcfd4b7

Reply:
To clarify, the `type` should still be `string` but have a `format => uri`.

Reply:
@ockham this bit.

Reply:
Oops, missed this one. On it!

Reply:
7f5db79e2b